.github/workflows/llm_performance_tests.yml

```diff
@@ -745,6 +745,34 @@ jobs:
           pip uninstall trl -y
           call conda deactivate
 
+      - name: Prepare igpu perf test for transformers 4.45 (32-32 int4+fp16)
+        shell: bash
+        run: |
+          sed -i 's/{today}_test4/{today}_test5/g' python/llm/dev/benchmark/all-in-one/run.py
+          sed -i "s/path to your local model hub/$MODEL_HUB_DIR/g" python/llm/test/benchmark/igpu-perf/32-32_int4_fp16_445.yaml
+
+      - name: Test on igpu for transformers 4.45 (32-32 int4+fp16)
+        shell: cmd
+        run: |
+          call conda activate igpu-perf
+          pip install transformers==4.45.0
+          pip install accelerate==0.33.0
+          pip install trl
+
+          set SYCL_CACHE_PERSISTENT=1
+          set BIGDL_LLM_XMX_DISABLED=1
+
+          cd python\llm\dev\benchmark\all-in-one
+          move ..\..\..\test\benchmark\igpu-perf\32-32_int4_fp16_445.yaml config.yaml
+          set PYTHONIOENCODING=utf-8
+          python run.py >> %CSV_SAVE_PATH%\32-32_int4_fp16\log\%LOG_FILE% 2>&1
+          if %ERRORLEVEL% neq 0 if %ERRORLEVEL% neq -1073740791 (exit /b 1)
+          python ..\..\..\test\benchmark\igpu-perf\check_csv_results.py --yaml-file config.yaml --suffix test5
+          if %ERRORLEVEL% neq 0 (exit /b 1)
+
+          pip uninstall trl -y
+          call conda deactivate
```

Comment:
We should downgrade accelerate to the default version after transformers 4.45 tests